### DISCUSSION TITLE
Rewrite

### DISCUSSION THREAD

**Discussion by Duued** [LeoCx1000/discord-bots#41] [General]
Hey everyone! I was thinking about contributing. 

I know you are rewriting the bot currently, my question is are you rewriting everything, or just putting things in functions and making them compatible with both standard commands and app_commands

Thanks, dogknife.

**Comment by Hadock-is-ok** [comment]
everything

**Reply by Hadock-is-ok** [reply]
Leo is currently in the middle of a hiatus so he can study, but this is a full duckbot rewrite.

**Reply by Duued** [reply]
So we're rewriting all of the commands? Even the ones that are perfectly okay?

**Reply by Hadock-is-ok** [reply]
You're not a part of the duck-bot github team, so id ask leo for access so you can see the todo list.

**Reply by tookender** [reply]
We are currently rewriting everything, yes, you can check the the todo list [here](https://github.com/orgs/DuckBot-Discord/projects/3) @Duued
